janus/lib/server/controllers/admin_controller.rb

```diff
@@ -173,7 +173,7 @@ def update_cc_agreement
       send_email(
         "Project Lead of #{project_name}",
         project.contact_email,
-        "A new guest user has agreed to the #{project_name} Code of Conduct",
+        "#{project_name} Code of Conduct was signed by #{@user.email}",
```

Comment:
Intentional? 😋

Reply:
Yes, because it was the small change we talked over in standup and probably isn't worth it's separate PR.